client/app/views/TrackViewer/ControlsContainer.vue

```diff
@@ -21,11 +21,11 @@ export default defineComponent({
 
   props: {
     lineChartData: {
-      type: Object as PropType<Ref<unknown>>,
+      type: Array as PropType<unknown[]>,
```

Comment:
Those components aren't written in typescript, nothing to export.

Reply:
I meant the data types that are used as inputs not the display components.  The input data has an interface in those files that can be used to substitute unknown?  This isn't really necessary because it is a pass through anyways.

Reply:
Sure, I think that would be out of scope in this PR but we can do it if you want.  I'd rather just convert the lienchart and eventchart to TS though.